vault_auth.sh

```diff
@@ -6,4 +6,4 @@ if [ "$VAULT_AUTH_GITHUB_TOKEN" = "" ]; then
     export VAULT_AUTH_GITHUB_TOKEN=${token}
 fi
 echo "Authenticating with the Vault"
-vault auth --method=github > /dev/null
```

Comment:
Probably out of this PR but we should allow `VAULT_ADDR` to be configurable probably?  Do you think that's in general bb8 scope, and if so should I raise a ticket?

Reply:
Yes good idea; feel free to raise a ticket!!